CompanionAppService/CompanionAppService.cs

```diff
@@ -647,6 +647,16 @@ public static Profile ProfileFromJson(JObject json, DateTime timestamp)
                 Profile.docked = (bool)json["commander"]["docked"];
                 Profile.alive = (bool)json["commander"]["alive"];
 
+                if (json["commander"]["capabilities"] != null)
```

Comment:
I know this might not be the best time to discuss this, but why aren't we using something like Newtonsoft DeserializeObject instead of parsing each single property?

Reply:
I'd be happy to go with that option too but that would require some more significant changes to our overall parsing approach than I wanted to implement at this time.

Reply:
Got it.